### PR TITLE
Concurrency check should include session.last_bookmarks

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

File: src/neo4j/_async/work/session.py [hunk 1]
```diff
@@ -353,6 +353,7 @@ async def last_bookmark(self) -> t.Optional[str]:
             return self._bookmarks[-1]
         return None
 
+    @AsyncNonConcurrentMethodChecker.non_concurrent_method
```
Comment (bigmontz) [inline src/neo4j/_async/work/session.py:356]: What does this decorator means?

Comment (robsdedude) [reply src/neo4j/_async/work/session.py:356]: It's coming from this PR https://github.com/neo4j/neo4j-python-driver/pull/989

The driver offers an optional debug mode where it tries to detect concurrent usage of objects that are not safe to use concurrently.

**Review by bigmontz — APPROVED** [review]
🔢

File: src/neo4j/_async/work/session.py (hunk 1 above)
Comment (bigmontz) [reply src/neo4j/_async/work/session.py:356]: thanks

**Pull request merged**